pkg/controller/ps/controller.go

```diff
@@ -347,23 +372,26 @@ func (r *PerconaServerMySQLReconciler) deleteCerts(ctx context.Context, cr *apiv
 			Name:      secretName,
 		}, secret)
 		if err != nil {
+			log.Info("Secret not found or already deleted", "name", secretName, "error", err.Error())
 			continue
 		}
 
 		err = r.Client.Delete(ctx, secret,
 			&client.DeleteOptions{Preconditions: &metav1.Preconditions{UID: &secret.UID}})
 		if err != nil {
+			log.Error(err, "Failed to delete secret", "name", secretName)
 			return errors.Wrapf(err, "delete secret %s", secretName)
 		}
+		log.V(1).Info("Secret deleted", "name", secretName)
```

Comment:
why this is debug level? if this should be debug level, why the other logs above are info level?